### PR TITLE
Added Embeddings Option on custom Triplets

### PR DESCRIPTION
**Pull request by abhiram1809**
# Description

Added an option to include embeddings for triplets that are upserted manually in the Knowledge Graph Index increasing their fetch-ability in the System

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [X] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: storage/default__vector_store.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "embedding_dict": {}, "text_id_to_ref_doc_id": {}, "metadata_dict": {} }
```
Comment (hatianzhang) [inline storage/default__vector_store.json:1]: do we need to commit those json files?

Comment (abhiram1809) [reply storage/default__vector_store.json:1]: No, they were created by make format and make lint for some reason, added automatically, so I did not touch them.

Comment (hatianzhang) [reply storage/default__vector_store.json:1]: could we remove them 
I don't think they are needed

Comment (abhiram1809) [reply storage/default__vector_store.json:1]: Ok sure

Comment (abhiram1809) [reply storage/default__vector_store.json]: Done @hatianzhang , please check

**Review by hatianzhang — APPROVED** [review]

**Comment by hatianzhang** [comment]
@abhiram1809 since we just upgrade to 0.10.*
could you help the pr. Basically move your changes to new files

**Comment by abhiram1809** [comment]
Thanks @hatianzhang , I got a bit busy

**Pull request merged**